packages/frontend/src/main/components/common/SpeckleViewer.vue

```diff
@@ -50,7 +56,11 @@ export default {
     this.$refs.rendererparent.appendChild(this.domElement)
 
     await this.viewer.unloadAll()
-
+    console.log(this.viewer)
+    console.log(this.noScroll)
+    if (this.noScroll) {
+      this.viewer.cameraHandler.controls.mouseButtons.wheel = 0
```

Comment:
Since the viewer instance is re-used each time the viewer is opened, you should also handle the inverse scenario - a viewer with scrolling turned off being used with scrolling enabled again. So you should explicitly enable scrolling if !noScroll

Reply:
Checked, and this is actually difficult to get going properly without deeper hacking with the camera controls lib used in the viewer. I'll add a guard to make sure this is happening only in embeds.